libraries/botbuilder/src/teamsActivityHandler.ts

```diff
@@ -86,6 +87,14 @@ export class TeamsActivityHandler extends ActivityHandler {
                 return await this.handleTeamsCardActionInvoke(context);
             } else {
                 switch (context.activity.name) {
+                    case 'config/fetch':
+                        return ActivityHandler.createInvokeResponse(
+                            this.handleTeamsBotConfigAuthFetch(context, context.activity.value)
```

Comment:
Why "Auth" in both handleTeamsBotConfigAuthFetch and handleTeamsBotConfigAuthSubmit?

Isn't that config/auth one of the request types?

![image](https://github.com/microsoft/botbuilder-js/assets/7191986/9ce0ad25-8a0e-497c-91e3-a9be383b93dc)

Reply:
I'll rename. @yingduyingdu As for tests, I haven't written them yet so they will not be passing.